tests/generation/test_utils.py

```diff
@@ -2140,12 +2140,26 @@ def _test_attention_implementation(self, attn_implementation):
                 del model_eager
                 gc.collect()
 
-                model_attn = model_class.from_pretrained(
-                    tmpdirname,
-                    torch_dtype=torch.float16,
-                    low_cpu_mem_usage=True,
-                    attn_implementation=attn_implementation,
-                ).to(torch_device)
+                # SDPA only: Some models partially support SDPA: they will load SDPA where they can by default, but if
```

Comment:
We still run the SDPA<>Eager `generate` test when there is partial SDPA support 🙌